add support for JOOL SIIT/NAT64 implementation

julianschuh opened the pull request:
this PR adds support for the SIIT/NAT64 implementation JOOL.

JOOL supports netfilter and iptables; for the iptables mode the targets JOOL and JOOL_SIIT are provided and corresponding iptables rules must be created.

You can find more information in the official documentation of JOOL:
 - https://www.jool.mx/en/run-vanilla.html
 - https://www.jool.mx/en/run-nat64.html

MaxKellermann commented:
Your PR contains text that is not present in the commit message.
Please add documentation.

julianschuh commented:
Thanks for your feedback. I added more information to the commit message and also added the JOOL and JOOL_SIIT targets to the documentation.

Thanks and best Regards
Julian
